docs/design/data-modeling.md

```diff
@@ -78,20 +78,33 @@ When selecting an `id` scheme:
 - An account and a transfer may share the same `id` — they belong to different "namespaces".
 - Avoid requiring a central oracle to generate each unique `id` (e.g. an auto-increment field in SQL).
   A central oracle may become a performance bottleneck when creating accounts/transfers.
+- Sequences of identifiers with long runs of strictly increasing values are amenable to very
```

Comment:
I'm not sure if we want to speak about internals like compaction at the data-modeling level of docs?